.packit.yaml

```diff
@@ -0,0 +1,59 @@
+# See the documentation for more information:
+# https://packit.dev/docs/configuration/
+packit_instances: ["stg"]
+upstream_project_url: https://github.com/fedora-copr/copr.git
+
+packages:
+
+  python:
+    downstream_package_name: python-copr
+    upstream_package_name: copr
+    paths: 
+      - ./python
+    specfile_path: python-copr.spec
+    files_to_sync: 
+      - python-copr.spec
+
+  cli:
+    downstream_package_name: copr-cli
+    upstream_package_name: copr-cli
+    paths: 
+      - ./cli
+    specfile_path: copr-cli.spec
+    files_to_sync: 
+      - copr-cli.spec
```

Comment:
Could we have `after: [ python ]` here, and perform `copr-cli` build after `python-copr` (that's good to resolve the dependencies).

Reply:
We have not yet a solution for this, at least I think (@lachmanfrantisek @csomh am I wrong?) .
But while working on it I will pay attention if any simple solution comes to my mind!

Reply:
Note there's the `--after-build-id` feature in `copr-cli` in case the problem is how to "chain" the builds Copr side.

Reply:
I really like the approach with `after: [ ... ]` like @praiskup suggested. People will typically want to specify the order of builds in monorepos.

Similar approach uses docker-compose with `depends_on` field.

As pavel mentioned, you can use build batches to solve this.

Reply:
@majamassarini is right, we haven't thought about this before. I'll put it on our list :) I would call it `build_after`, though, b/c that would be more explicit.